web/ASC.Web.Client/src/components/pages/Settings/categories/common/team-template.js

```diff
@@ -0,0 +1,716 @@
+          guestsCaption: false,
+        },
+      });
+    }
+    this.setState({
+      showReminder: false,
+      isChanged: false,
+      id: customNames.id,
+      selectedOption: {
+        key: 0,
+        label: customNames.name,
+      },
+    });
+  };
+  render() {
+    const { t, isLoading } = this.props;
+
+    const {
+      userCaption,
+      usersCaption,
+      groupCaption,
+      groupsCaption,
+      regDateCaption,
+      groupHeadCaption,
+      userPostCaption,
+      guestCaption,
+      guestsCaption,
+      selectedOption,
+      formErrors,
+      isChanged,
+      showReminder,
+      availableOptions,
+      isLoadingData,
+    } = this.state;
+
+    return isLoading ? (
+      <Loader className="pageLoader" type="rombs" size="40px" />
+    ) : (
+      <StyledComponent>
+        <FieldContainer
+          className="team-template_field-container main-field-container"
+          isVertical
+          labelText={`${t("Template")}:`}
+          place="top"
+        >
+          <ComboBox
+            options={availableOptions}
+            selectedOption={selectedOption}
+            onSelect={this.onCustomSchemaSelect}
+            isDisabled={isLoadingData}
+            noBorder={false}
+            scaled={true}
+            scaledOptions={true}
+            dropDownMaxHeight={300}
+            className="team-template_text-input"
+          />
+        </FieldContainer>
+
+        <FieldContainer
+          className="team-template_field-container"
+          isVertical
+          labelText={`${t("User")}:`}
+          place="top"
+          hasError={formErrors.userCaption}
+          errorMessage={`${t("EmptyFieldError")}`}
+        >
+          <TextInput
+            name={"userCaption"}
+            className="team-template_text-input"
+            scale={true}
+            value={userCaption}
+            hasError={formErrors.userCaption}
+            onChange={this.onChangeInput}
+            isDisabled={isLoadingData}
+            placeholder={t("AddName")}
+          />
+        </FieldContainer>
+
+        <FieldContainer
+          className="team-template_field-container"
+          isVertical
+          labelText={`${t("Users")}:`}
+          place="top"
+          hasError={formErrors.usersCaption}
+          errorMessage={`${t("EmptyFieldError")}`}
+        >
+          <TextInput
+            name={"usersCaption"}
+            className="team-template_text-input"
+            scale={true}
+            value={usersCaption}
+            hasError={formErrors.usersCaption}
+            onChange={this.onChangeInput}
+            placeholder={t("AddName")}
+            isDisabled={isLoadingData}
+          />
+        </FieldContainer>
+
+        <FieldContainer
+          className="team-template_field-container"
+          isVertical
+          labelText={`${t("Group")}:`}
+          place="top"
+          hasError={formErrors.groupCaption}
+          errorMessage={`${t("EmptyFieldError")}`}
```

Comment:
Replace to errorMessage={t("EmptyFieldError")}

Reply:
Replaced.